### DISCUSSION TITLE
EXCITED FOR GITHUB EXTERNSHIP 2.0

### DISCUSSION THREAD

**Discussion by khushi9871** [github/india#582] [Students]
WHEN CAN WE START APPLYING TO RESGITERED ORGANISATION

**Comment by Siddhant-K-code** [comment]
The schedule will be out soon, you can start applying after the positive review of your profile.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
